playbooks/centos7_fips.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
The playbook can just be named `fips.yml` I think.

Reply:
Done with the rebase and file name change.